Content.Server/GameObjects/Components/Power/WirePlacerComponent.cs

```diff
@@ -23,6 +23,14 @@ internal class WirePlacerComponent : Component, IAfterInteract
         /// <inheritdoc />
         public override string Name => "WirePlacer";
 
+        private string _wirePrototypeID;
+
+        public override void ExposeData(ObjectSerializer serializer)
+        {
+            base.ExposeData(serializer);
+            serializer.DataField(ref _wirePrototypeID, "wirePrototypeID", "HVWire");
```

Comment:
I don't know that prototype IDs are the sanest way to go about this. PJB may have more to say.

Reply:
I'm not sure how else to handle it - any suggestions?

Reply:
I'm not sure, personally. A proper enum for the different wire types seems like a better fit - since it seems like that's what this is actually being used for. That way, we can have different wire prototypes that still have the same wire type enum value.

Reply:
I don't see what's wrong with just having this be a prototype ID, honestly.

Reply:
My main concern was that this gets used in line 51 for checking prototype equality, which to me seems like almost an abuse of prototype IDs - since the actual code is trying to avoid creating multiple wires of the same type on the same tile.

Reply:
I understand your concern now, and have recoded the section that checks for a wire on a tile. Instead of comparing prototype ID, it looks for Nodes of a yaml-specified NodeGroupID set.

Reply:
I have implemented a WireType enum which is checked instead of the prototype ID.